main/modes/system/intro/introMode.c

```diff
@@ -1060,14 +1070,14 @@ static void introDrawSwadgeTouchpad(int64_t elapsedUs, vec_t touchPoint, list_t*
             x -= (TOUCHPAD_RADIUS * 3 * getSin1024(a)) / (1024 * 4);
         }
         int32_t y = TOUCHPAD_Y - (TOUCHPAD_RADIUS * 3 * getCos1024(a)) / (1024 * 4);
-        drawCircleFilled(x, y, 4, c115);
+        drawCircleFilled(x, y - 5, 4, c555); //emily DO NOT HARDCODE THIS -5 GOOFBALL TODO
```

Comment:
I got Cunningham's Law'd and redrew the gem. Make sure to `git pull` before continuing work here.

Reply:
(note, that's an SVG if you want to download it)